modules/microshift-install-rhel-image-mode-build-image.adoc

```diff
@@ -0,0 +1,115 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-rhel-image-mode.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-rhel-image-mode-build-image_{context}"]
+= Building the bootc image
+
+Build your Red Hat Enterprise Linux as a bootable container image by using a `ContainerFile` and include {microshift-short}.
```

Comment:
Can we rephrase this? Let's make it more explicit that the image we build includes MicroShift.